tests/general/CPDBearTest.py

```diff
@@ -50,3 +50,14 @@ def test_bad_file(self):
         result = list(self.uut.run_bear_from_section([], {}))
 
         self.assertNotEqual(result, [])
+
+    def test_unsupported_language(self):
+        self.section.update_setting(
+            key='language', new_value='unsupported_language')
+
+        self.uut = CPDBear({'file_name': 'hello world  \n'},
+                           self.section,
+                           self.queue)
+
+        list(self.uut.run_bear_from_section([], {}))
+        self.assertEqual(self.uut.message_queue.queue[0].log_level, 40)
```

Comment:
I see you couldn't find where to get that constant number from. It seems you can get it from `logging`:

```python
>>> import logging
>>> logging.ERROR
40
```